### ISSUE TITLE
Bug found

### ISSUE THREAD

**Issue by Suppen2** [capnkirok/Voxels#42]
Hi i found a bug
When  add this mod all my other items named to unnamed thats crazy. 
Can you help me ?

**Comment by HariboTer** [comment]
Yup, I got this issue too after throwing together a lot of mods. After tediously figuring out that Voxels was what turned all mod item names into "Unnamed [semirandomNumber]", I luckily found out that changing a config setting to true like this:

B:"Hide Voxel Values When Searching"=true

fixes the issue. A workaround for whose existence I'm very glad because I really would have hated throwing out Voxels (whose gimmicky setting I somewhat adore) again because I have no clue about most of the other 10k items I'm playing with :D
